test/controllers/hostgroups_controller_test.rb

```diff
@@ -38,6 +38,23 @@ def test_clone
     assert_template 'new'
   end
 
+  def test_copies_facets
```

Comment:
I think this test will belong with all facet behavior tests: https://github.com/theforeman/foreman/blob/a9b9b7d9a2caacb0062e7d4863f26b3ceaa87e22/test/unit/facet_test.rb#L237
This test file already has the proper setup.